kubernetes driver // allow custom annotations and labels (#1938)
docker/buildx: docs/reference/buildx_create.md

Passes additional driver-specific options.

> Note: When using quoted values for example for the `nodeselector`, `annotations` or

Reviewer: ```suggestion
Note: When using quoted values for example for the `nodeselector`, `annotations`, `labels` or
```

Author: I was expecting this comment :)

Should I remove `for example` from the sentence, since it's supposed to be all-inclusive?

```suggestion
Note: When using quoted values for the `nodeselector`, `annotations`, `labels` or
```

Author: (note: I applied my suggestion)

- `limits.memory` - Sets the limit memory value specified in bytes or with a valid suffix. Example `limits.memory=500Mi`, `limits.memory=4G`
- `serviceaccount` - Sets the created pod's service account. Example `serviceaccount=example-sa`
- `"nodeselector=label1=value1,label2=value2"` - Sets the kv of `Pod` nodeSelector. No Defaults. Example `nodeselector=kubernetes.io/arch=arm64`

> - `"annotations=domain/thing1=value1,domain/thing2=value2"` - Sets additional annotations on the deployments and pods. No Defaults. Example `annotations=mycorp.io/owner=sarah`

Reviewer 2: ```suggestion
- `"annotations=domain/thing1=value1,domain/thing2=value2"` - Sets additional annotations on the deployments and pods. No Defaults. Example `annotations=example.com/owner=sarah`
```

> - `"labels=domain/thing1=value1,domain/thing2=value2"` - Sets additional labels on the deployments and pods. No Defaults. Example `labels=domain.com/team=rd`

Reviewer 2: ```suggestion
- `"labels=domain/thing1=value1,domain/thing2=value2"` - Sets additional labels on the deployments and pods. No Defaults. Example `labels=example.com/team=rd`
```
